### ISSUE TITLE
Use Rails ActionView helpers in views

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#483]
Switch `<label>` and `<input>` tags over to Rails view helpers. For example switch `<label for="email"><%= _('Email') %></label>` to `<%= f.label :email %>`

**Comment by briri** [comment]
@xsrust noted on the latest PR that using the Rails helpers for labels prevents those labels from being translated. We should ensure that all labels are using the standard html tags `<label for="guidance_published"><%= _('Published') %></label>`. Input fields can use the helpers still.

**Comment by briri** [comment]
FYI. the `forms.scss` file will now automatically apply a colon to the end of any `<label>` that does not have the `no-colon` or `checkbox-label` class (e.g. `<label><%= _('Published') %></label>` results in: `Published:` and `<label class="no-colon"><%= _('Published') %></label>` results in: `Published`). 

It will also add an asterisk at the end of your labels if you give it the `required` class (e.g. `<label class="no-colon"><%= _('Published') %></label>` results in: `Published * :`).

**Comment by stephaniesimms** [comment]
Closing this issue pending next round of accessibility testing.
